sources/packages/web/src/components/generic/Banner.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <v-alert
+    class="v-alert--variant-contained-text"
```

Comment:
This class seems to be defining something that is not aligned with the `variant="outlined"` also defined. Can you please just elaborate on why it was needed?

Reply:
When this class is applied, it renders the alert with a background color too. I used it so that I will not have to define and use a custom css class to make the alert have a background color.  The reason for doing this is that when variant="outlined", the alert does not have a background color. When variant="contained-text", the alert will have a background color with no border. There is currently to variant that renders the alert with background color and border. FYI: The class also uses the color specified. E.g if we set color="warning", the background color of the alert also changes to orange. Same with error.
![image](https://user-images.githubusercontent.com/72251620/169355492-d8f39e59-8a13-4b1c-ab5b-fcc075ba4ed9.png)

Reply:
Did you mean this? 
![image](https://user-images.githubusercontent.com/77353155/169390843-f15f5153-3146-4636-9ce5-5c41e7ec8073.png)i think no need to specify the class, you can give ` variant="contained-text"`

Reply:
This is the result of variant="contained-text" 
<img width="946" alt="image" src="https://user-images.githubusercontent.com/72251620/169393707-ec60ffa1-bb11-41b5-a295-c46efcc97323.png">
 
No border. If we all agree to use this then I can change easily. @andrewsignori-aot  @dheepak-aot @guru-aot Your thoughts ?

Reply:
mm, I see, ` variant="contained-text"` is without border and and `variant="outlined"` is without background color
![image](https://user-images.githubusercontent.com/77353155/169396926-02b81d81-7f40-4763-b054-bf4bbef265b5.png)
![image](https://user-images.githubusercontent.com/77353155/169397055-769fc5ae-e7cf-458e-b4e4-04edc745e726.png)
and v-alert `border` property gives as 
![image](https://user-images.githubusercontent.com/77353155/169397148-92ee3797-8402-4096-960a-e3f4ba2f1640.png)
![image](https://user-images.githubusercontent.com/77353155/169397199-5dd8a637-4f18-43b1-8c47-b0f84b810407.png)
I think, you can check with @hellolynn-tbtb and decide